api/src/libs/gh-utils/gh-helpers.js

```diff
@@ -46,3 +47,40 @@ export const jsonReader = (jsonData, paths = null) => {
   }
   throw Error(`Cannot process data for the path: ${paths}.`);
 };
+
+/**
+ * Validate and filter PR:
```

Comment:
maybe a better description of what the fn does? Not really sure what you mean by filtering a pr